### PR TITLE
Research Assistant

### LINKED ISSUE #13752: Science needs an intern role
## Description

Just like engineering, medical, and security: science needs a role for new players to learn the ins-and-outs.

Possible names:
- science intern
- research assistant

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

adds an intern role for the science department.
Closes #13752 

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added the research assistant role, for all the new people wanting to get into the science department.

### CODE REVIEW

File: Resources/Locale/en-US/job/job-names.ftl [hunk 1]
```diff
@@ -4,6 +4,7 @@ job-name-cadet = Security Cadet
 job-name-hos = Head of Security
 job-name-detective = Detective
 job-name-scientist = Scientist
+job-name-research-assistant = research assistant
```
Comment (crazybrain23) [inline Resources/Locale/en-US/job/job-names.ftl:7]:
```suggestion
job-name-research-assistant = Research Assistant
```
This should probably be capitalised the same as the other jobs.

Comment (crazybrain23) [inline Resources/Locale/en-US/job/job-names.ftl:7]:
```suggestion
job-name-research-assistant = Research Assistant
```
This should probably be capitalised the same as the other jobs.

File: Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml [hunk 2]
```diff
@@ -70,6 +82,7 @@
     - state: default
     - state: idintern-cadet
 
+
```
Comment (crazybrain23) [inline Resources/Prototypes/Entities/Objects/Misc/identification_cards.yml:85]: Remove the extra line break

**Review by crazybrain23 — CHANGES_REQUESTED** [review]
Just a couple nitpicks, but otherwise good.

File: Resources/Locale/en-US/job/job-names.ftl (hunk 1 above)
Comment (EmoGarbage404) [reply Resources/Locale/en-US/job/job-names.ftl:7]: this is lowercase as this is being changed with #13769. I didn't want to create extra work with it.

**Review by crazybrain23 — APPROVED** [review]

**Pull request merged**